src/main/frontend/components/settings.cljs

```diff
@@ -191,11 +191,18 @@
        [:input#home-default-page.form-input.is-small.transition.duration-150.ease-in-out
         {:default-value secs
          :on-blur       (fn [event]
-                          (when-let [value (-> (util/evalue event)
-                                               util/safe-parse-int)]
-                            (when (< 0 value (inc 600))
-                              (state/set-state! [:electron/user-cfgs :git/auto-commit-seconds] value)
-                              (ipc/ipc "userAppCfgs" :git/auto-commit-seconds value))))}]]]]))
+                          (let [value (-> (util/evalue event)
+                                          util/safe-parse-int)]
+                            (if (and (number? value)
+                                     (< 0 value (inc 600)))
```

Comment:
I am also curious why we set the maximum interval?

Reply:
CC @tiensonqin
I personally prefer another approach: when max diff-lines is over xxx lines

Reply:
I'd also appreciate a different approach or a (way higher) maximum.
_For people using Logseq throughout the day for small notes, a 10-min-commit is literally flooding someones git-activity._

Reply:
@seemayr Thanks. Created a new discussion for this! Welcome to share ideas.

https://github.com/logseq/logseq/discussions/4928

This Pull request is only a temp fix to roll back to a legal state when setting value is not accepted.